src/utils/credentials.ts

```diff
@@ -164,3 +164,19 @@ export async function getAndVerifyFullCredentials() {
   spinner.stop();
   return credentials;
 }
+
+export function getAndVerifyCredentials() {
```

Comment:
It seems like the other function is more specific to accessing retoolDB so just made another one to verify Retool credentials. lmk if that's ok @PeteTheHeat